### PR TITLE
Added my canvas art

### PR DESCRIPTION
**Pull request by AashiGoel**
# Zero To Mastery - Canvas 2022 🎉🎉

## PR contains:
My Art was made using a `<canvas>` tag in HTML. I added an image of a jack o' lantern to the canvas with a spooky text on the top using google fonts.

## Breaking changes
None

## Screenshots
icon

### CODE REVIEW

File: src/art/AashiGoel/Halloween.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nosifer">
+    </head>
+<body>
+<h2>Happy Halloween!!!</h2>
+<img src="Halloween.jpg" alt="pumpkin">
```
Comment (sophiabrandt) [inline src/art/AashiGoel/Halloween.html:9]: Please provide the license to the image. Images are usually under copyright, so we need to make sure that we can showcase this without violating copyright.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: src/art/AashiGoel/Halloween.html (hunk 1 above)
Comment (AashiGoel) [reply src/art/AashiGoel/Halloween.html:9]: I downloaded this image from google and I can't find the exact image on searching now.....

Comment (AashiGoel) [reply src/art/AashiGoel/Halloween.html:9]: https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.cre8iveskill.com%2Fshop%2Fqueen-mummy-vector-art&psig=AOvVaw3HcBVWYOBiowS2F9r9b-8K&ust=1666113269633000&source=images&cd=vfe&ved=0CA0QjRxqFwoTCMjgnN_h5_oCFQAAAAAdAAAAABAE
This is the image I downloaded

Comment (sophiabrandt) [reply src/art/AashiGoel/Halloween.html:9]: Downloading images from Google doesn't give you copyright.

I suggest that you replace the image with something from unsplash, which has images with permissive licenses.

Comment (AashiGoel) [reply src/art/AashiGoel/Halloween.html:9]: okay I will :)

**Pull request closed without merging**